users/vieting/experiments/switchboard/ctc/feat/network_helpers/specaug_configurable.py

```diff
@@ -201,5 +292,90 @@ def get_masked():
     return x
 
 
-def get_specaug_funcs() -> list:
-    return [sort_filters_by_center_freq, _mask, random_mask, transform]
+def transform_with_variance(data, network, **config):
```

Comment:
Please check a diff between this and the existing `transform` function. It does not seem like this is already finished. Parts of the original function are missing and it is not yet possible to use a combination of the existing and the new behavior.

Reply:
It is merged now